Makefile

```diff
@@ -13,7 +13,7 @@ fmt:
 test-unit:
 	@go test -short ./...
 
-test-system: build
+test-system: install
```

Comment:
If you're going to do this, you should get rid of everything related to the dist directory, including all the PATH stuff in pkg/test. And I think that deserves a separate PR.

Reply:
this was a misguided attempt to do something else.  removed the commit.

Reply:
👍